includes/header.php

```diff
@@ -47,36 +47,36 @@
   <meta name="author" content="Phil Ewels">
   <link rel="shortcut icon" href="/assets/img/logo/nf-core-logo-square.png" type="image/png" />
   <link rel="alternate" type="application/rss+xml" title="nf-core: Events" href="/events/rss" />
-  <link href="/assets/css/Chart.min.css" rel="stylesheet">
   <!-- FontAwesome -->
   <script src="https://kit.fontawesome.com/471b59d3f8.js"></script>
   <!-- Global site tag (gtag.js) - Google Analytics -->
   <script async src="https://www.googletagmanager.com/gtag/js?id=UA-68098153-2"></script>
   <!-- Other JS -->
-  <script src="/assets/js/jquery.min.js"></script>
-  <script src="/assets/js/bootstrap.bundle.min.js"></script>
-  <script src="/assets/js/highlight.min.js"></script>
+  <script src="/assets/lib/jquery.min.js"></script>
+  <script src="/assets/lib/bootstrap.bundle.min.js"></script>
+  <script src="/assets/lib/highlight.min.js"></script>
   <!-- Page-specific CSS and JS -->
   <?php if (isset($import_moment) && $import_moment) : ?>
-    <script src="/assets/js/moment.js"></script>
-    <script src="/assets/js/moment-timezone-with-data-10-year-range.js"></script>
+    <script src="/assets/lib/moment.js"></script>
+    <script src="/assets/lib/moment-timezone-with-data-10-year-range.min.js"></script>
   <?php endif;
   if (isset($import_leaflet) && $import_leaflet) : ?>
-    <link href="/assets/css/leaflet.css" rel="stylesheet">
-    <link href="/assets/css/leaflet.fullscreen.css" rel="stylesheet">
-    <script src="/assets/js/leaflet.js"></script>
-    <script src="/assets/js/Leaflet.fullscreen.min.js"></script>
+    <link href="/assets/lib/leaflet.css" rel="stylesheet">
+    <script src="/assets/lib/leaflet.js"></script>
+    <!-- NOT ON NPM --> <link href="/assets/css/leaflet.fullscreen.css" rel="stylesheet"> <!-- NOT ON NPM -->
+    <!-- NOT ON NPM --> <script src="/assets/js/Leaflet.fullscreen.min.js"></script> <!-- NOT ON NPM -->
```

Comment:
If I see it correctly, this is only used in the for the contributors map on the community page and adds the ability to show the map in full-screen. What is the use case for that function or could we maybe just drop this dependency?

Reply:
Yeah maybe let's drop it. I had used it in a previous project so it was an easy drop-in. But I guess rarely used in this case 👍🏻